src/shared/components/Modal/Modal.module.scss

```
@@ -46,14 +46,14 @@
   &__header {
     &__title {
       margin: 12px 10px 6px 10px;
-      @include m.text-style-extended('xl', 700, orange-600);
+      @include m.text-style-extended('xl', 700, gray-900);
```

Stylelint (17.2.0)

[error] Expected empty line before at-rule (at-rule-empty-line-before)

(at-rule-empty-line-before)